pytplot/tplot.py

```diff
@@ -194,16 +193,37 @@ def tplot(name,
             exporter.export(save_png)
 
         if display:
+            # Set up all things needed for when a user asks to save plot from window
+            layout_orig = QtPlotter.generate_stack(name, var_label=var_label, auto_color=auto_color,
+                                                   combine_axes=combine_axes,
+                                                   mouse_moved_event=pytplot.hover_time.change_hover_time,
+                                                   crosshair=crosshair)
```

Comment:
Should this crosshair be set to the tplot_options one?  

Does it even matter what this one is set to if the plot is static like this?  Could we get rid of the crosshair parameter tplot()?